types/moduleClients.go

```diff
@@ -0,0 +1,11 @@
+package types
```

Comment:
Why CamelCase for the filename? Can we use `module_clients.go`?

Reply:
Ah sorry! 🙊